data/json/items/armor/storage.json

```diff
@@ -1258,6 +1258,7 @@
     "warmth": 10,
     "material_thickness": 2,
     "flags": [ "BELTED", "WATER_FRIENDLY" ],
+    "use_action": [ { "type": "attach_molle", "size": 8 }, { "type": "detach_molle" } ],
     "armor": [ { "encumbrance": [ 5, 20 ], "coverage": 70, "covers": [ "torso" ], "specifically_covers": [ "torso_hanging_back" ] } ]
```

Comment:
Gonna have to change this from [5, 20] to 5 with a volume encumber modifier like above. most bags hover around something like .15 right now.

Reply:
It is now [5,33]+16 for tucked/carabiner pockets.